adapter/redisquota/config/config.proto

```diff
@@ -0,0 +1,35 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package adapter.redisquota.config;
+
+import "google/protobuf/duration.proto";
+
+option go_package = "config";
+
+message Params {
+    // Minimum number of seconds that deduplication is possible for a given operation.
+    google.protobuf.Duration min_deduplication_duration = 1;
+
+    // Redis network address
+    string redis_server_url = 2;
+
+    // Socket type for talking to redis
+    string socket_type = 3;
```

Comment:
Should this be an enum? Or should the comment say something about what this is expected to be?